shell/platform/darwin/macos/framework/Source/FLETextInputPlugin.mm

```diff
@@ -72,7 +72,7 @@ - (instancetype)initWithViewController:(FLEViewController*)viewController {
   if (self != nil) {
     _flutterViewController = viewController;
     _channel = [FlutterMethodChannel methodChannelWithName:kTextInputChannel
-                                           binaryMessenger:viewController
+                                           binaryMessenger:viewController.binaryMessenger
```

Comment:
can we just do viewController.engine.binaryMessenger here? (assuming the lifecycle sequence is right)

Reply:
I think we could.  I'm not too confident in the change considering I don't think we have alot of test coverage on this code.  I'd prefer doing the simplest, slightest correct to get what we want.